lms/djangoapps/courseware/courses.py

```diff
@@ -224,14 +224,15 @@ def get_course_about_section(course, section_key):
                 except Exception:  # pylint: disable=broad-except
                     html = render_to_string('courseware/error-message.html', None)
                     log.exception(
-                        u"Error rendering course={course}, section_key={section_key}".format(
-                            course=course, section_key=section_key
-                        ))
+                        u"Error rendering course=%s, section_key=%s",
```

Comment:
You can view by commit which pylint violation I'm squashing. This one is logging-not-lazy or logging-format-interpolation. https://bitbucket.org/logilab/pylint/pull-request/169/add-new-warning-logging-format/diff

The reasoning here is that log is a special function that can take arguments to interpolate into the log message. If you pass your message as arguments, eg `log.warn('message %s', arg)`, the formatting interpolation will *only* occur if the log message is actually hit and evaluated. However, with `.format()`, the message will always be interpolated even if the `log` isn't actually ever hit (for example, if the log level is different). So it's a slight performance advantage.

We could choose to not follow this one, but right now we are.